src/thesis.tex

```diff
@@ -790,11 +790,11 @@
     
     \includegraphics[width=\textwidth] {imgs/screenshots/screen_accounts_browse.png}
     
-    Similar view exists in the users tab to present all created user accounts in the system.
+    The similar view exists in the user's tab to present all created user accounts in the system.
```

Comment:
it's a tab about users, not a tab owned by a user

Reply:
fixed